src/human.py

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright 2018 Sveta Haas.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+#
+# Refer to the README and COPYING files for full details of the license
+#
+class Human(object):
+    def __init__(self,Age,Name):
```

Comment:
Please separate between the arguments with a space and name arguments with lowercase.
Arguments or variables should be lowercase, only classes use camel-case. Constants use uppercase.